Makefile

```diff
@@ -0,0 +1,3 @@
+all: headless-test
+headless-test:
+	xvfb-run -a bundle exec rake test
```

Comment:
Yeah, this is a place where things verge on unnecessary complexity. I'm using xvfb-run to start up a headless X session that the browser instance running the integration tests can connect to. The Makefile should probably go, and it would probably be more conventional to use the headless gem rather than manually invoke xvfb like this - I was just going with what I already know.